nx/public/sl/components.js

```diff
@@ -52,6 +52,7 @@ class SlSelect extends LitElement {
     name: { type: String },
     label: { type: String },
     value: { type: String },
+    disabled: { type: Boolean },
```

Comment:
I'd love to spend time figuring out a more sustainable way of doing something of these (like the input has the spread), but this is good.